tensorflow/python/ops/variable_scope.py

```diff
@@ -692,6 +698,39 @@ def _get_single_variable(self,
     return v
 
 
+  # Initialize variable when no initializer provided
+  def _get_default_initializer(self, name, shape=None, dtype=dtypes.float32):
+    """Provide a default initializer and a corresponding value.
+
+    Args:
+      name: see get_variable.
+      shape: see get_variable.
+      dtype: see get_variable.
+
+    Returns:
+      initializer and initializing_from_value. See get_variable above.
```

Comment:
I think initializing_from_value isn't documented in get_variable. So perhaps clarify that here?